### ISSUE TITLE
Stretched splashscreen in default project

### ISSUE THREAD

**Issue by LoranRendel** [ionic-team/capacitor#6597]
### Capacitor Version

```
Latest Dependencies:       

  @capacitor/cli: 5.0.3    
  @capacitor/core: 5.0.3   
  @capacitor/android: 5.0.3
  @capacitor/ios: 5.0.3    

Installed Dependencies:    

  @capacitor/ios: not installed
  @capacitor/cli: 5.0.3        
  @capacitor/android: 5.0.3
  @capacitor/core: 5.0.3
```

### Platform(s)

Using default project created by `pnpm create @capacitor/app`. Splashscreen is stretched.
Screen resolution is 1080x2400.

![image](https://github.com/ionic-team/capacitor/assets/7853801/9b7f4122-7c03-4236-8d7f-9ad0334d3e0e)

**Comment by jcesarmobile** [comment]
This is a limitation of the androidx splash screen support library that doesn't play well with full screen splash anymore, they recommend using an icon as it does on Android 12 and newer.

We kept the full screen splash on the default template despite it's stretched on some devices because we have the `.show()` method that shows the splash again and in that case it can't show the icon splash screen.

You can disable the support library like this:
https://capacitorjs.com/docs/apis/splash-screen#android-12-splash-screen-api

**Comment by LoranRendel** [comment]
> This is a limitation of the androidx splash screen support library that doesn't play well with full screen splash anymore, they recommend using an icon as it does on Android 12 and newer.
> 
> We kept the full screen splash on the default template despite it's stretched on some devices because we have the `.show()` method that shows the splash again and in that case it can't show the icon splash screen.
> 
> You can disable the support library like this: https://capacitorjs.com/docs/apis/splash-screen#android-12-splash-screen-api

Sorry, I haven't specified Android version, it's 10 in my case. 
I've changed parent to `AppTheme.NoActionBar`, the splash screen is still stretched.

**Comment by jcesarmobile** [comment]
Yeah, the support library is used in Android 11 and older, on Android 12 and newer it will use an icon instead of the full screen image, the app icon by default, it's not possible to use a full screen image on Android 12 and newer.

If you have disabled the support library, then you should be able to use the configuration options from https://capacitorjs.com/docs/apis/splash-screen#configuration
to try to adjust the image to the screen, you can use `androidScaleType` to try different scalings, or `layoutName` to use a custom layout xml file. Or use 9-patch images.

But if you don't use `SplashScreen.show()` in your code, I would recommend not disabling the support library and switch to using an icon instead of a full screen image, it's the only thing that will fit on all screens without stretching. 
https://developer.android.com/develop/ui/views/launch/splash-screen
